doc/guides/tips-tricks-and-troubleshooting.md

```diff
@@ -0,0 +1,77 @@
+# Tips, Tricks & Troubleshooting
+
+## Rules of thumb
+
+<details>
+<summary><strong>Provide motivation for the change</strong></summary>
```

Comment:
It's valid markdown 
https://github.com/refack/node/blob/6b3170cea8dc8ad78a687e20e7e859cd91fc79ed/doc/guides/tips-tricks-and-troubleshooting.md